Makefile

```diff
@@ -173,17 +173,11 @@ clean-all:
 # Stdlib modules that we repackage as standalone packages
 
 # TODO: also include test directories included in other stdlib modules
-build/test.data: $(CPYTHONLIB) node_modules/.installed
-	./tools/file_packager.sh build/test.data --js-output=build/test.js \
-		--preload $(CPYTHONLIB)/test@/lib/python$(PYMAJOR).$(PYMINOR)/test
-	npx terser build/test.js -o build/test.js
+build/test.tar: $(CPYTHONLIB) node_modules/.installed
+	cd $(CPYTHONLIB) && tar -cvf $(PYODIDE_ROOT)/build/test.tar test
```

Comment:
Should we exclude __pycache__ just in case?

Reply:
Sure, done.